### ISSUE TITLE
Document the Generation of the Confidencen and Traversability Output 

### ISSUE THREAD

**Issue by JonasFrey96** [leggedrobotics/wild_visual_navigation#161]
## node.prediction
 in update_prediction of the traversability_estimator:
1. WVN Node
```
out_trav = mission_node.prediction[fs, 0]
```
The model prediction for channel 0 contains a sigmoid layer therefore the traversability output is restricted to 0-1.

2. Traversability Estimator (update_prediction)
```
node.prediction = self._last_trained_model(data)
```

## node.confidence 
1. WVN Node
```
out_conf = mission_node.confidence[fs]
```
2. Traversability Estimator (`update_prediction`)
```
reco_loss = F.mse_loss(node.prediction[:, 1:], node.features, reduction="none").mean(dim=1)
node.confidence = self._traversability_loss._confidence_generator.inference_without_update( reco_loss )
```
Uses the confidence running mean boundaries that determine the confidence in the prediction. 
`update_prediction` in both cases is called when the image is added to the graph in the image_callback.
So actually everything seems reasonable for me.
